### DISCUSSION TITLE
deploy public path

### DISCUSSION THREAD

**Discussion by ac615223s5** [nuxt/nuxt#16191] [Questions] [answered]
i got a warning not to put /public/ in the path when linking to the public folder, but when i do not include /public/ it does not work when building. how should i fix this?

**Comment by TheAlexLichter** [comment, marked as answer]
Linking to files in public should work via `/`. To access e.g. `/public/my-logo.png`, you can write `<img src="/my-logo.png">`.

**Comment by Gringo675** [comment]
Yes, but when your baseUrl is not default "/", it doesn't work. For example, if `baseURL: '/test/'` the file in '/public/img/logo.png' will be available from address 'http://localhost:3000/test/img/logo.png', but in app `<img src="/img/logo.png" >` creates link to 'http://localhost:3000/img/logo.png', and we have 404 error. So, what to do in that case? Maybe write another path in config? Can't figure out.

**Reply by TheAlexLichter** [reply]
Feel free to raise an issue ☺️

**Reply by Gringo675** [reply]
done https://github.com/nuxt/framework/issues/9583
